src/context.cc

```diff
@@ -1,10 +1,34 @@
 #include <locale.h>  /* locale support    */
 #include "context.h"
 #include <iostream>
+#include <unistd.h>
 using namespace v8;
 
 Nan::Persistent<Function> ContextWrapper::constructor;
 
+static gpgme_error_t passphrase_cb (void *opaque, const char *uid_hint, const char *passphrase_info, int last_was_bad, int fd) {
+  int res;
+  char pass[255];
```

Comment:
I fear a nice buffer overflow here.

Reply:
Hmm makes sense, its copying from opaque but not checking the length. I will fix it.

Reply:
Fixed!